css/style.css

```diff
@@ -11,7 +11,7 @@ body {
  font-family: "arial";
  font-size:0.99em;
  margin: 15px;
- padding: 0
+ padding: 0;

```

Comment:
This is a bugfix. It could have been in a separate patch/pull request (OK to have it here imo, just a suggestion for next patches)